services/jupyterhub/Pipfile

```diff
@@ -0,0 +1,19 @@
+[[source]]
```

Comment:
Are you including the `Pipfile` and `Pipfile.lock` in the commit on purpose? If so, don't you want to use them in teh `Dockerfile`?

Reply:
Oops thanks. I did use pipenv to have the packages in pycharm.